base/_mixins.scss

```diff
@@ -103,155 +146,351 @@
 	}
 }
 
-// Column Grid
-// Simple mixin to display the grid - options are either 2, 3, or 4
-// @param number of columns
-// @usage column-grid(2)
-// @johncionci
-@mixin column-grid( $columns: null ) {
-	@if $columns == 3 {
-		@include span-columns(4);
-	} @else if $columns == 4 {
-		@include span-columns(3);
-	} @else {
-		@include span-columns(6);
-	}
-}
 
-// List Reset
-// Remove the margin, padding, and bullets from lists
-// @johncionci
-//
+/// list-reset() =
+/// Remove the margin, padding, and bullets from lists
+/// @author jcionci
+///
+/// @param {None}
+///
+/// @example scss - Usage
+///   // ul{ @include list-reset(); }
+///
+/// @example css - Output
+///   // ul{ list-style-type: none; margin: 0; padding: 0; }
```

Comment:
the properties and their values should probably be on their own line (to be consistent with the touch-hover() example output)

Same with line 180 & 195